CONTRIBUTING.md

```diff
@@ -241,7 +237,12 @@ checker, and leave out unnecessary detail:
 * use `float` instead of `Union[int, float]`.
 
 Some further tips for good type hints:
-* avoid invariant collection types (`List`, `Dict`) in argument
+* use built-in generics (`list`, `dict`, `tuple`, `set`), instead
+  of importing them from `typing`, **except** for arbitrary length tuples
+  (`Tuple[int, ...]`);
```

Comment:
It would be helpful here to link to the mypy issue, so we can easily refer back to it.